src/pages/Login.vue

```diff
@@ -189,93 +241,15 @@ export default {
     },
     routeAfterLogin() {
       // route user to the relevant page after login is complete
-      if (this.redirectTo != "") {
+      if (this.redirectTo == "") {
         // there is no other page to redirect the user to
         // redirect to the home page
         this.$router.push({ name: "Home" });
+      } else {
+        // redirect to the relevant page with its params
+        this.$router.push({ name: this.redirectTo, params: this.redirectParams });

```

Comment:
```suggestion
        this.$router.replace({ name: this.redirectTo, params: this.redirectParams });

```

Reason - `push` adds the URL to the browser's history stack. And after logging in, if I press the back button, the url changes from `localhost:8080` to `localhost:8080/#/login`

It is not breaking anything but the expected behaviour is that once they are logged in, they should not be able to press back and go back to the login ka url